<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProjectView</class>
 <widget class="QDialog" name="CreateProjectView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>581</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="projectDescriptField">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>541</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dialogTitleLbl">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateBtn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createBtn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>170</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Project Description:</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>252</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Project status:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="projectStatusLE"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Project Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="projectNameLE"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>254</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Max team size:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="maxTeamSizeLE"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Min team size:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="minTeamSizeLE"/>
    </item>
   </layout>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>projectNameLE</zorder>
  <zorder>line</zorder>
  <zorder>maxTeamSizeLE</zorder>
  <zorder>projectStatusLE</zorder>
  <zorder>minTeamSizeLE</zorder>
  <zorder>projectDescriptField</zorder>
  <zorder>label_3</zorder>
  <zorder>dialogTitleLbl</zorder>
  <zorder>updateBtn</zorder>
  <zorder>createBtn</zorder>
  <zorder>label_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
